### PR TITLE
[test]|[documentation] Test local build documentation

### PR DESCRIPTION
**Pull request by arjkesh**
*Issue #, if available:*
#265 
## Checklist
- [x] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [build] | [test] | [build, test] | [ec2, ecs, eks, sagemaker]

*Description:*
- Added an integration test for local documentation to protect us against issues like #265
- Currently added to EC2 tests since it spins up an EC2 instance, but it actually doesn't test DLC on EC2, so I'm happy to move it to sanity tests
- Considering skipping this in PR context since it re-does an entire build

*Tests run:*
- Testing in draft PR

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,7 +53,7 @@ For the purposes of testing in your personal account, the following managed perm
     ``` 
 2. Login to ECR
     ```
-    aws ecr get-login-password --region us-west-2 | docker login --username AWS --password-stdin $ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com
```
Comment (arjkesh) [inline README.md:56]: replace us-west-2 with $REGION

File: test/dlc_tests/ec2/test_README.py [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+    github_repo_name = ctx.run("basename `git rev-parse --show-toplevel`").stdout
+    with ec2_connection.prefix(f"git clone {github_repo_link}", hide=True):
+        with ec2_connection.cd(github_repo_name):
+            ec2_connection.run(f"git checkout {commit_hash}")
+            ec2_connection.run(f"export ACCOUNT_ID={account_id} && "
+                               f"export REGION={region} && "
+                               f"export REPOSITORY_NAME=beta-{framework}-training", hide=True)
+            # Log into ECR
+            ec2_connection.run("aws ecr get-login-password --region $REGION | docker login --username AWS "
+                               "--password-stdin $ACCOUNT_ID.dkr.ecr.us-west-2.amazonaws.com", hide=True)
```
Comment (arjkesh) [inline test/dlc_tests/ec2/test_README.py:32]: replace us-west-2 with $REGION

**Comment by YYStreet** [comment]
Are we still needing this one? @arjkesh

**Comment by arjkesh** [comment]
Closing for now as this is stale

**Pull request closed without merging**